examples/post_training_quantization/openvino/llm_compression/main.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Will it automatically be added to the regular Examples CI check?
cc @alexsu52 @ksilligan

Reply:
To tell the truth, I planed to add a test but didn't find the test suite for examples.

Reply:
@AlexKoff88  tests for examples https://github.com/openvinotoolkit/nncf/blob/develop/tests/cross_fw/examples/test_examples.py

And i think link to this example can be added to https://github.com/openvinotoolkit/nncf?tab=readme-ov-file#demos-tutorials-and-samples

Reply:
Added a test for the example